src/engraving/libmscore/slur.h

```diff
@@ -65,14 +65,17 @@ class SlurSegment final : public SlurTieSegment
 class Slur final : public SlurTie
 {
     void slurPosChord(SlurPos*);
+    int _sourceStemArrangement = -1;
```

Comment:
transient property needed to remember stem arrangement of a slur copied to the clipboad